packages/thirdweb/src/exports/thirdweb.ts

```diff
@@ -282,3 +282,27 @@ export {
   type VerifyTypedDataParams,
   verifyTypedData,
 } from "../auth/verify-typed-data.js";
+
+// ------------------------------------------------
+// Indexer
+// ------------------------------------------------
+export {
+  getBlock,
+  type GetBlockParams,
+} from "../indexer/endpoints/getBlock.js";
+export {
+  getLatestBlockNumber,
+  type GetLatestBlockNumberParams,
+} from "../indexer/endpoints/getLatestBlockNumber.js";
+export {
+  getNFTsByCollection,
```

Comment:
should just be injected inside 

extensions/erc721/getNFTs
extensions/erc1155/getNFTs